app/jobs/set_allele_registry_ids.rb

```diff
@@ -8,7 +8,7 @@ def perform
       if allele_registry_id == '_:CA'
         v.allele_registry_id = 'unregistered'
         v.save
-      else
+      elsif not allele_registry_id.nil?
```

Comment:
You if you keep this a negative condition you might want to switch to `!` instead of `not`, otherwise you could swap it to `if allele_registry_id.present?`